app/main.py

```diff
@@ -1,8 +1,47 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int
+                 ) -> None:
```

Comment:
bad code style, fix it here too. Here is an example: https://github.com/mate-academy/py-linter-formatter/blob/master/checklist.md#code-style

Reply:
But in my code I had done that:

class Car:
    # write your code here
    pass
    def __init__(
            self,
            comfort_class: int,
            clean_mark: int,
            brand: str
    ) -> None: